include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -772,12 +772,12 @@ class argument_parser
 
                 arg = argv[i];
 
-                if (arg == "1")
+                if (arg == "true")
                     version_check_user_decision = true;
-                else if (arg == "0")
+                else if (arg == "false")
                     version_check_user_decision = false;
                 else
-                    throw validation_error{"Value for option --version-check must be 1 or 0."};
+                    throw validation_error{"Value for option --version-check must be true or false."};
```

Comment:
I would allow both `0` and `false`, otherwise this is an API break (do our bool options allow both `false` and `0`)?